tower-balance/Cargo.toml

```diff
@@ -22,10 +22,14 @@ categories = ["asynchronous", "network-programming"]
 edition = "2018"
 publish = false
 
+[features]
+log = ["tracing/log"]
+default = []
```

Comment:
IMO, the `log` feature should _probably_ be on by default, given that most users currently expect `tower` to emit `log` records, and removing it will break them...